ignite/contrib/metrics/precision_recall_curve.py

```diff
@@ -1,21 +1,10 @@
-from typing import Any, Callable, Tuple
+from typing import Callable
 
 import torch
 
 from ignite.metrics import EpochMetric
 
 
-def precision_recall_curve_compute_fn(y_preds: torch.Tensor, y_targets: torch.Tensor) -> Tuple[Any, Any, Any]:
```

Comment:
@sayantan1410 I think you misunderstood my point that we discussed offline. This PR wont solve DDP issue. Please revert the change. The problem is not here but in the way how works `EpochMetric.compute` method with unusual outputs. Check in details how works https://github.com/pytorch/ignite/blob/master/ignite/contrib/metrics/average_precision.py and see its tests, for example: https://github.com/pytorch/ignite/blob/493d86dc0eae7ad1fb4fd703cff2e3c7230dfbbf/tests/ignite/contrib/metrics/test_average_precision.py#L160-L185

Try to understand how it works and why we do https://github.com/pytorch/ignite/blob/493d86dc0eae7ad1fb4fd703cff2e3c7230dfbbf/tests/ignite/contrib/metrics/test_average_precision.py#L177-L178

Next you can get back to precision_recall_curve and see that `precision_recall_curve_compute_fn` does not return a scalar as does AP function: https://github.com/pytorch/ignite/blob/493d86dc0eae7ad1fb4fd703cff2e3c7230dfbbf/ignite/contrib/metrics/average_precision.py#L8

This is the root of the problem.

Reply:
@vfdev-5 Okay so the problem is that precision_recall_curve returns 3 arrays but compute_fn for epoch metric can process an array !!